clients/restore/restore.go

```diff
@@ -146,13 +150,17 @@ func (c Client) fullRestore(ctx context.Context, path string) error {
 		Execute(); err != nil {
 		return fmt.Errorf("failed to restore KV snapshot: %w", err)
 	}
-	log.Println("INFO: Restoring SQL snapshot")
-	if err := c.PostRestoreSQL(ctx).
-		ContentEncoding("gzip").
-		ContentType("application/octet-stream").
-		Body(sqlBytes).
-		Execute(); err != nil {
-		return fmt.Errorf("failed to restore SQL snapshot: %w", err)
+
+	// TODO: Should we have some way of wiping out any existing SQL on the server-side in the case when there is no backup?
```

Comment:
This kind of situation seems like it would be a little messy, but I would tend to think it would be safer to not delete any existing data. If anything, we would probably want a Y/N prompt so the user doing the restore could decide if, when restoring a 2.0.x backup to a 2.1+ server, they want to wipe out existing sql data or not. It would require some server-side work as well...and it seems like it wouldn't come up very often. Something to keep in mind I guess if it happens more frequently than I am imagining.